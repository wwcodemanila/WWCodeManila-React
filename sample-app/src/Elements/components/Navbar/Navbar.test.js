import React from 'react';
import { shallow } from 'enzyme';

import Navbar from './Navbar';

test('renders proper', () => {
  const component = shallow(<Navbar />)
  expect(component).toMatchSnapshot()
})
