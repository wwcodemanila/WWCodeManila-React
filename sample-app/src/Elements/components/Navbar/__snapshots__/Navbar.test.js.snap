// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders proper 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Navbar />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "aria-label": "main navigation",
      "children": Array [
        <NavbarBrand />,
        <NavbarMenu
          menuList={
            Array [
              Object {
                "link": "/",
                "menuLabel": "Home",
              },
              Object {
                "link": "/about",
                "menuLabel": "About React",
              },
              Object {
                "link": "/references",
                "menuLabel": "References",
              },
            ]
          }
        />,
      ],
      "className": "navbar",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "menuList": Array [
            Object {
              "link": "/",
              "menuLabel": "Home",
            },
            Object {
              "link": "/about",
              "menuLabel": "About React",
            },
            Object {
              "link": "/references",
              "menuLabel": "References",
            },
          ],
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "aria-label": "main navigation",
        "children": Array [
          <NavbarBrand />,
          <NavbarMenu
            menuList={
              Array [
                Object {
                  "link": "/",
                  "menuLabel": "Home",
                },
                Object {
                  "link": "/about",
                  "menuLabel": "About React",
                },
                Object {
                  "link": "/references",
                  "menuLabel": "References",
                },
              ]
            }
          />,
        ],
        "className": "navbar",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "menuList": Array [
              Object {
                "link": "/",
                "menuLabel": "Home",
              },
              Object {
                "link": "/about",
                "menuLabel": "About React",
              },
              Object {
                "link": "/references",
                "menuLabel": "References",
              },
            ],
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
